clouddriver-aws/src/main/groovy/com/netflix/spinnaker/clouddriver/aws/deploy/handlers/BasicAmazonDeployHandler.groovy

```diff
@@ -116,9 +117,14 @@ class BasicAmazonDeployHandler implements DeployHandler<BasicAmazonDeployDescrip
 
       def regionScopedProvider = regionScopedProviderFactory.forRegion(description.credentials, region)
 
-      def loadBalancers = lookupHelper().getLoadBalancersByName(regionScopedProvider, description.loadBalancers)
+      def loadBalancers = loadBalancerLookupHelper().getLoadBalancersByName(regionScopedProvider, description.loadBalancers)
```

Comment:
where did we land on handling the case of someone passing an ALB name as the load balancer name?

it looks like if we were going to continue to let that work that the targetGroupArns that get set below needs to be a union of loadBalancers.targetGroupArns + targetGroups.targetGroupARNs